src/components/TokenTranslationUpdate/TokenTranslationUpdate.md

```diff
@@ -0,0 +1,33 @@
+### Token Translation Updation
+
+This component will help to update the token Translation
+
+```js
+import { useState, useEffect } from 'react';
+import TokenTranslationUpdate from './TokenTranslationUpdate';
+
+const [tokenDetail, setTokenDetail] = React.useState({
+  token: 'સત્યમાં',
+  occurrences: [
+    {
+      sentenceId: 65001004,
+      offset: [12, 19],
+    },
+    {
+      sentenceId: 65001003,
+      offset: [34, 41],
+    },
+    {
+      sentenceId: 65001001,
+      offset: [13, 20],
+    },
+  ],
+  translations: {},
+});
+
+const projectId = 100000;
```

Comment:
we can avoid hardcoding the projectId, token as this may not always exist in the demo server, instad we can show a project dropdown, and a token dropdown for that project, show first project and first token by default